bioinfokit/visuz.py

```diff
@@ -681,7 +682,7 @@ def mhat(df="dataframe", chr=None, pv=None, log_scale=True, color=None, dim=(6,4
         else:
             ylm = np.arange(0, max(df['tpval']+1), 1)
         ax.set_yticks(ylm)
-        ax.set_xticklabels(xlabels, rotation=ar)
+        ax.set_xticklabels(xlabels, fontsize=axtickfontsize, fontproperties=axtickfontname, rotation=ar)
```

Comment:
This enables the caller to change font name and font size of tick labels, which is especially useful when the labels is in another language.